model/properties.go

```diff
@@ -10,4 +10,5 @@ type PropertyRepository interface {
 	Get(id string) (string, error)
 	Delete(id string) error
 	DefaultGet(id string, defaultValue string) (string, error)
+	// TODO: GetAll() (map[string]string, error) may be useful
```

Comment:
This is usually private info and should not be exposed as metrics.

Reply:
I'm not 100% sure with this, it's not just a generic repository, it's repository of properties with <10 entries in my production instance.

But I removed this in d4cd72c26410778558467c76360be7d6e8f64af0 and used another approach for metrics, more optimal, as I believe

Reply:
Yes, it is a generic repository, and currently there's only about 5 entries in this table, 3 of them are secrets:

- InitialSetup: time Navidrome was setup
- LastScan: time when the last scan was run
- JWTSecret: Used to sign JWT tokens
- LastFMSessionKey: Session key received after authenticating with Last.fm (per user)
- ListenBrainzSessionKey: Session key received after authenticating with ListenBrainz (per user)

We may add things like UI preferences in the future, but even then, I'm not sure if this kind of information counts as "metrics"